components/src/AppHeaderNavigation/MockHttpResponses/navigation_data_response.json

```diff
@@ -0,0 +1,1051 @@
+{
+	"self": {
+		"type": "elasticpath.collections.links",
+		"uri": "/?zoom=navigations:element,navigations:element:child,navigations:element:child:child,navigations:element:child:child:child,navigations:element:child:child:child:child,navigations:element:child:child:child:child:child,navigations:element:child:child:child:child:child:child,navigations:element:child:child:child:child:child:child:child,navigations:element:child:child:child:child:child:child:child:child",
+		"href": "https://vestri-us.epdemos.com/cortex/?zoom=navigations:element,navigations:element:child,navigations:element:child:child,navigations:element:child:child:child,navigations:element:child:child:child:child,navigations:element:child:child:child:child:child,navigations:element:child:child:child:child:child:child,navigations:element:child:child:child:child:child:child:child,navigations:element:child:child:child:child:child:child:child:child"
```

Comment:
Any reason why we didn't use http://reference.epdemos.com?

Reply:
No reason here.  Arbitrarily used vestri-us.epdemos.com.  I can change it to use https://reference.epdemos.com for consistency though